Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -4,6 +4,10 @@
 
 #include "Schema/UnrealObjectRef.h"
 
+#include <improbable/c_worker.h>
+
+using Worker_EntityId_Key = int64;
```

Comment:
Why the `_key`?

Reply:
Because this typedef is only used for entity id map keys, as these don't compile on linux. This is to give more context than what you would get with just an int64

Reply:
But the key is just the entity id right?

Reply:
Yes but due to clang, worker_entityid expands to long which won't compile with TMap.

Reply:
Ah I see 👍